client/my-sites/hosting/style.scss

```diff
@@ -1,14 +1,5 @@
 @import "calypso/assets/stylesheets/shared/mixins/breakpoints";
 
-.preview-pane__content {
-	.hosting {
-		margin: 0;
-		padding: 0;
-		overflow-y: auto;
-		max-height: calc(100vh - 300px);
```

Comment:
It looks like this might have been neccessary for mobile. it's not scrolling properly on mobile
<img width="653" alt="Screenshot 2024-05-07 at 4 25 13 pm" src="https://github.com/Automattic/wp-calypso/assets/22446385/8243919a-4973-4fb3-9c74-bcc73e8359f7">